File: fabric/src/main/java/cn/breadnicecat/lovemod/fabric/LoveModImpl.java
package cn.breadnicecat.lovemod.fabric;

import cn.breadnicecat.lovemod.LoveMod;
import net.fabricmc.api.ModInitializer;

public class LoveModImpl implements ModInitializer {
	@Override
	public void onInitialize() {
		LoveMod.LOGGER.info("Loading Fabric");
		LoveMod.init();
	}
}

File: fabric/src/main/kotlin/cn/breadnicecat/lovemod/fabric/LoveModImpl.kt
package cn.breadnicecat.lovemod.fabric

import cn.breadnicecat.lovemod.LoveMod
import net.fabricmc.api.ModInitializer

class LoveModImpl : ModInitializer {
    override fun onInitialize() {
        LoveMod.LOGGER.info("Loading Fabric")
        LoveMod.init()
    }
}
